src/main/java/it/gov/pagopa/pu/classification/model/view/ClassificationView.java

```diff
@@ -0,0 +1,267 @@
+package it.gov.pagopa.pu.classification.model.view;
+
+  import com.fasterxml.jackson.annotation.JsonProperty;
+  import jakarta.persistence.Entity;
+  import jakarta.persistence.Id;
+  import jakarta.persistence.IdClass;
+  import jakarta.persistence.Table;
+  import lombok.AllArgsConstructor;
+  import lombok.Data;
+  import lombok.EqualsAndHashCode;
+  import lombok.NoArgsConstructor;
+
+  import java.io.Serializable;
+  import java.time.LocalDate;
+  import java.time.LocalDateTime;
+
+  @Entity
+  @Table(name = "classification")
+  @Data
+  @AllArgsConstructor
+  @NoArgsConstructor
+  @EqualsAndHashCode(callSuper = false)
+  @IdClass(ClassificationView.class)
```

Comment:
this annotation is needed onlu in case of composite id, as I can see, this model has just one ID field